jbdevstudio/multiple/jbdevstudio-multiple.target

```diff
@@ -44,11 +39,6 @@
       <unit id="org.apache.commons.io" version="2.0.1.v201105210651"/>
       <unit id="org.apache.commons.codec" version="1.3.0.v201101211617"/>
       <unit id="org.apache.commons.httpclient" version="3.1.0.v201012070820"/>
-      <unit id="org.hamcrest" version="1.1.0.v20090501071000"/>
-      <unit id="org.hamcrest.core" version="1.1.0.v20090501071000"/>
-      <unit id="org.hamcrest.text" version="1.1.0.v20090501071000"/>
```

Comment:
Whyis it only included in the JBDS TP then? What in JBDS needs it? And why doesn't that same thing also need it in JBT? 

It's here: http://download.jboss.org/jbosstools/targetplatforms/jbosstoolstarget/4.50.0.Beta1-SNAPSHOT/REPO/plugins/ as a result of SWTBot adding version 2 plugins at version 1.3, not 1.1 stuff - https://github.com/jbosstools/jbosstools-target-platforms/blob/4.50.x/jbosstools/multiple/jbosstools-multiple.target#L444-L445 )

In JBDS, we have 6 hamcrest plugins, mostly from 1.1. But for some reason the 1.3 core plugin appears in addition to the 1.1 one. 

https://devstudio.redhat.com/targetplatforms/jbdevstudiotarget/4.50.0.Beta1-SNAPSHOT/REPO/plugins/ (different IU list, different versions)